MomohNobert/BasecampMock
# Handle the collaborators' resource

#### What does this PR do?
This PR adds the collaboration resource to the BaseCamp Project

#### Description of Task to be completed?
- added feature for deleting collaborators
- added feature for adding collaborators
- checked for edge cases

#### How should this be manually tested?
 - clone repository
- startup terminal
- cd into the cloned repository
- ensure rails is properly installed
- run ```gem install bundler```
- run ```bundle install```
- start the server by typing ```rails server``` into the terminal

#### Any background context you want to provide?
- User must be must be logged in and must have a project created to test this feature

#### Screenshots (if appropriate)
None

#### Questions:
None
